app/assets/javascripts/chf_image_viewer.js

```diff
@@ -438,13 +438,6 @@ jQuery(document).ready(function($) {
       event.preventDefault();
       var id = this.getAttribute('data-member-id');
       chf_image_viewer().show(id);
-
-      // GA
-      _gaq.push(['_trackEvent',
-           'ImageViewer',
-           'fileSetId',
-           id
-          ]);
     });
```

Comment:
You replaced this with the data attributes on the link?  If that works properly, that is better! I was worried it wouldn't because of the fancy JS stuff being done in to trigger this click, instead of using an ordinary hyperlink, so instead of doing the right thing and testing and getting it right, I was just lazy and did something I knew would work.